co.posit.rstudio.yaml

```diff
@@ -0,0 +1,59 @@
+id: co.posit.rstudio
+runtime: org.freedesktop.Sdk
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --env=ELECTRON_TRASH=gio
+  - --filesystem=host
+
+modules:
+  - name: R
+    buildsystem: autotools
+    config-opts:
+      - --enable-R-shlib
+    sources:
+      - type: archive
+        url: https://cran.r-project.org/src/base/R-4/R-4.4.1.tar.xz
+        sha256: fbd755314a5cc08c57aabff839ff791fb43a9052a2c6524ec3be96075fd34dde
+
+  - name: wrapper
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/opt/rstudio/rstudio --enable-features=UseOzonePlatform --ozone-platform-hint=auto "$@"
+        
+  - name: RStudio
+    buildsystem: simple
+    build-commands:
+      - install -d /app/opt
+      - cp -r ./ /app/opt/rstudio
+      - install -d /app/share/icons/hicolor/512x512/apps/
+      - cp rstudio.png /app/share/icons/hicolor/512x512/apps/co.posit.rstudio.png
+      - install -d /app/share/metainfo
+      - cp co.posit.rstudio.metainfo.xml /app/share/metainfo/
+      - install -d /app/share/applications
+      - cp co.posit.rstudio.desktop /app/share/applications/
+    sources:
+      - type: archive
+        url: https://download1.rstudio.org/electron/rhel9/x86_64/rstudio-2024.09.0-375-x86_64-fedora.tar.gz
+        sha256: 8521d15205457edea9ea259ecf50a216189aa52ab790d27bffd84aecdf22231d
+      - type: file
+        path: co.posit.rstudio.metainfo.xml
+      - type: file
+        path: co.posit.rstudio.desktop
+      - type: file
+        url: https://github.com/rstudio/rstudio/blob/main/src/node/desktop/resources/freedesktop/icons/512x512/rstudio.png?raw=true
```

Comment:
this should be from a commit not `main`, if hash changes build will fail.

Reply:
I changed the ref to `v2024.09.0+375`